sdcm/nemesis.py

```diff
@@ -353,10 +359,11 @@ def get_list_of_methods_by_flags(
             run_with_gemini: Optional[bool] = None,
             networking: Optional[bool] = None,
             kubernetes: Optional[bool] = None,
-            limited: Optional[bool] = None) -> List[str]:
+            limited: Optional[bool] = None,
+            topology_changes: Optional[bool] = None) -> List[str]:
         attributes = locals()
         flags = {flag_name: attributes[flag_name] for flag_name in
-                 ['disruptive', 'run_with_gemini', 'networking', 'kubernetes', 'limited'] if attributes[flag_name] is not None}
+                 self.flag_list if attributes[flag_name] is not None}
```

Comment:
@aleksbykov , let's get rid of flag_list and do just following:
```suggestion
        subclasses_list = self._get_subclasses(
            disruptive=disruptive,
            run_with_gemini=run_with_gemini,
            networking=networking,
            kubernetes=kubernetes,
            limited=limited,
            topology_changes=topology_changes
        )
```